satpy/tests/reader_tests/test_netcdf_utils.py

```diff
@@ -54,7 +54,6 @@ def get_test_content(self, filename, filename_info, filetype_info):
             - '/attr/global_attr'
             - 'dataset/attr/global_attr'
             - 'dataset/shape'
-            - 'dataset/dimensions'
```

Comment:
Looks like you are reverting changes done recently here.

Reply:
same as earlier, `netcdf_utils`in thi PR is not from the latest Satpy-master.